docs/concepts/what-is-filecoin.md

```diff
@@ -0,0 +1,27 @@
+---
+title: What is Filecoin?
+---
+
+# What is Filecoin?
+
+**Filecoin is a peer-to-peer network that stores files on the internet, with built-in economic incentives to ensure files are stored reliably over time.**
+
+In Filecoin, users pay storage miners to store their files. Storage miners are computers responsible for storing files and proving they have stored the files correctly over time. Anyone who wants to store their files or get paid for storing other users’ files can join Filecoin. Available storage and pricing is not controlled by any single company. Instead, Filecoin facilitates open markets for storing and retrieving files that anyone can participate in.
+
+![Diagram showing a four-step process for storing data with Filecoin. First step: User pays miner to store their data. Second step: Miner commits publicly via Filecoin’s blockchain to storing the data. Third step: The network constantly verifies that miners are storing data correctly. Fourth step: User pays miner to retrieve data.](./images/what-is-filecoin.png)
+
+## For users
+
+Filecoin provides users assurance their files have been stored safely and will be available in the future.
+
+Users can choose their preferred trade-off between cost, redundancy and speed by selecting the particular miners which store their files, or let software decide on their behalf. Applications that implement Filecoin can negotiate storage with any miner on the network. Unlike centralized storage systems, there is no need to implement a different API for each provider.
+
+At any time, users can verify that their files are being stored correctly by looking at proofs on Filecoin’s blockchain. This blockchain also powers Filecoin’s payment network and token, which users pay for storage with. Users don’t have to deal with the friction or fees associated with centralized payment networks.
+
+## For storage providers
+
+Filecoin enables storage providers to sell their storage on an open market.
+
+Storage providers are people and organizations that run miners, earning Filecoin tokens for doing so. A miner could be any internet-connected computer with spare disk space, or a dedicated system with lots of storage built specifically for Filecoin. Filecoin’s blockchain rewards miners for storing files, not for completing arbitrary proof-of-work computations.
```

Comment:
> Filecoin’s blockchain rewards miners for storing files, not for completing arbitrary proof-of-work computations.

- Let's take this sentence 1 step further and say something about usefulness of file storage — maybe "rewards miners for contributing useful storage to the internet" or something like that.

Reply:
Great call, I went with:

> Filecoin’s blockchain rewards miners for contributing useful storage to the internet, not for completing wasteful proof-of-work computations.